modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
+
+[role="_abstract"]
+When upgrading to a version >= 4.6 from a version < 4.6, you must patch the sensor and admission-control deployments  to set the `POD_NAMESPACE` environment variable.
+
+[NOTE]
+====
+If you are using Kubernetes, use `kubectl` instead of `oc` for the commands listed in this procedure.
+====
+
+.Procedure
+
+. Patch sensor to ensure `POD_NAMESPACE` is set:
++
+[source,terminal,subs=attributes+]
+----
+$ oc -n stackrox patch deployment sensor --type=json -p '[{"op":"add","path":"/spec/template/spec/containers/0/env/-","value":{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"fieldPath":"metadata.namespace"}}}}]'
+----
+. Patch admission-control to ensure `POD_NAMESPACE` is set:
```

Comment:
How about Collector?

Reply:
We don't currently use GetPodNamespace() in collector, nor do we set the `POD_NAMESPACE` var for collector elsewhere. Extending it to collector can be discussed, but currently there is no usecase.

Reply:
How about local scanner?

Reply:
Currently `POD_NAMESPACE` is only used in sensor and admission-control (through my newly added function). I think it's also used in central in some other context, but it's already set for central deployments.